link-checker.js

```diff
@@ -286,9 +326,7 @@ module.exports = function(directory, options = {}, callback) {
 		await Promise.all(localParentLinksArray.map(target => {
 			return new Promise((resolve, reject) => {
 				fileCounter += 1
-				fs.exists(path.resolve(directory, target), result => {
-					resolve(result)
-				})
+				resolve(fs.existsSync(path.resolve(directory, target)));
```

Comment:
fs.exists() is deprecated, but fs.existsSync() is not ([see here](https://nodejs.org/api/fs.html#fs_fs_existssync_path))

Reply:
👍

Reply:
If this case we don't need to wrap it into a promise anymore.

Reply:
So we could simplify it here to 

```js
localParentLinksArray.map(target => {
  fileCounter += 1
  return fs.existsSync(path.resolve(directory, target))
})

Reply:
This series of blocks can probably be simplified even further with the use of `fs.[...]Sync`, and as `fileCounter` is not used in the subsequent promise handling, the separate map blocks can probably be turned into just a single mapping pass, but that's probably better done in follow-up than as part of this PR.